examples/groups/group.yaml

```diff
@@ -2,11 +2,12 @@ apiVersion: groups.gitlab.crossplane.io/v1alpha1
 kind: Group
 metadata:
   name: example-group
+  # annotations:
+  #   crossplane.io/external-name: "11879378"

```

Comment:
I think for this situation this code make sence because you will not be able create group use Gitlab api. You just can import group or get group from exist at gitlab